rocAL/rocAL_pybind/examples/image_comparison.py

```diff
@@ -0,0 +1,148 @@
+        logging.error('Pass atleast 2 arguments (python image_comparison.py <ref_output_folder_path> <rocal_ouput_folder_path>')
+        exit()
+
+    # Open the two images
+    ref_output_path = sys.argv[1]
+    rocal_output_path = sys.argv[2]
+
+    if not (os.path.exists(ref_output_path) and os.path.exists(rocal_output_path)):
+        logging.error("Path does not Exists")
+        exit()
+
+    total_case_count = 0
+    passed_case_count = 0
+    failed_case_count = 0
+    failed_case_list = []
+    golden_output_dir_list = os.listdir(ref_output_path)
+    rocal_output_dir_list = os.listdir(rocal_output_path)
+    randomized_augmentation = ["Snow", "Rain", "Jitter", "SNPNoise"]
+    golden_file_path = ""
+    for aug_name in rocal_output_dir_list:
+        temp = aug_name.split('.')
+        file_name_s = temp[0].split('_')
+        if(len(file_name_s) > 3):
+            file_name_s.pop()
+            golden_file_path = "_".join(file_name_s) + ".png"
+        else:
+            golden_file_path = aug_name
+
+        #For randomized augmentation
+        if(file_name_s[0] in randomized_augmentation):
+            total_case_count=total_case_count+1
+            augmentation_name = aug_name.split('.')[0]
+            logging.info("Running %s",augmentation_name)
+            passed_case_count=passed_case_count+1
+            logging.info("PASSED ")
+        elif golden_file_path in golden_output_dir_list:
+            total_case_count=total_case_count+1
+            ref_file_path = ref_output_path+golden_file_path
+            rocal_file_path = rocal_output_path+aug_name
+            if(os.path.exists(rocal_file_path) and os.path.exists(ref_file_path)):
+                logging.info("Running %s ",aug_name.split('.')[0])
+                img1 = Image.open(ref_file_path)
+                img2 = Image.open(rocal_file_path)
+
+                # Check if the images have the same dimensions
+                if img1.size != img2.size:
+                    logging.info("The images have different sizes.")
+                    exit()
+
+                # Compare the pixel values for each image
+                pixeldiff = None
+                tot_count = 0
+                if 'larger' in aug_name:
+                    resize_width = 400
+                    resize_height = 300
+                    image_offset = 400
+                    pixeldiff, tot_count = compare_pixels(img1, img2, aug_name, resize_width, resize_height)
+                    pixeldiff2, tot_count2 = compare_pixels(img1, img2, aug_name, resize_width, resize_height, image_offset)
+                    pixeldiff = [x + y for x, y in zip(pixeldiff, pixeldiff2)]
+                    tot_count = tot_count + tot_count2
+                elif 'smaller' in aug_name:
+                    resize_width = 533
+                    resize_height = 400
+                    image_offset = 2400
+                    pixeldiff, tot_count = compare_pixels(img1, img2, aug_name, resize_width, resize_height)
```

Comment:
some names have this format pixeldiff and some tot_count

Use snake_case everywhere